src/App.js

```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] All components should be class-based. Kindly refactor your app component to meet this requirement.